.github/dependabot.yml

```
@@ -11,6 +11,16 @@ updates:
       - dependency-name: '*'
         update-types: ['version-update:semver-major']
 
+  - package-ecosystem: 'npm'
+    directory: '/api/documentation'
+    schedule:
+      interval: 'daily'
+      time: '09:00'
```

```suggestion
      time: '07:00'
```

Histoire de ne pas encombrer la CI